docs/src/main/sphinx/release/release-409.md

```diff
@@ -0,0 +1,86 @@
+# Release 409 (3 Mar 2023)
+
+## General
+
+* Add support for dropping nested fields with a `DROP COLUMN` statement. ({issue}`15975`)
+* Improve output of `EXPLAIN` queries to show statistics when a query contains
+  aggregations. ({issue}`16201`)
+* Improve performance of queries with distinct aggregations using table
+  statistics. This can be configured with the `optimizer.mark-distinct-strategy`
+  configuration property, and the `optimizer.use-mark-distinct` configuration
+  property is now deprecated in favor of it. ({issue}`15927`)
+* Remove the `distributed-index-joins-enabled` configuration property and
+  related functionality. ({issue}`15375`)
+* Fix failure when using non-comparable and non-sortable values as part of a
+  `VALUES` expression. ({issue}`16242`)
+
+## BigQuery connector
+
+* Add support for inserting default values. ({issue}`16327`)
+* Fix failure when non-lowercase column names exist in the `query` table
+  function. ({issue}`16075`)
+
+## ClickHouse connector
+
+* Remove support for the `ru.yandex.clickhouse.ClickHouseDriver` legacy JDBC
+  driver in the `clickhouse.legacy-driver` configuration property. ({issue}`16188`)
+* Remove support for specifying expressions in the `sample_by` table property to
+  prevent SQL injection. ({issue}`16261`)
+
+## Delta Lake connector
+
+* Automatically infer required Delta Lake version when creating new tables or
+  enabling new table features. ({issue}`16310`)
+* Fix query failure when reading Parquet files generated by Kafka Connect. ({issue}`16264`)
+
+## Hive connector
+
+* Add support for the Hadoop `DefaultCodec` to Hive formats. ({issue}`16250`)
+* Add a native CSV file format reader and writer. These can be disabled with the
```

Comment:
"native" is a little misleading. Should we say "optimized", "dedicated", "custom", etc.? cc @dain

Reply:
The configuration properties are all named native, so if 'native' is misleading, we probably need to rename all of those.